ui2/src/pages/custody/New.tsx

```diff
@@ -48,19 +70,40 @@ const NewComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> =
         s.payload.customer === party
     );
     if (!service) return;
-    const request: RequestOpenAccount = {
-      accountId: {
-        signatories: {
-          textMap: { [service.payload.provider]: {}, [service.payload.customer]: {} },
-        },
-        label: accountName,
-        version: '0',
-      },
-      observers: [publicParty],
-      ctrls: [service.payload.provider, service.payload.customer],
-    };
-    await ledger.exercise(Service.RequestOpenAccount, service.contractId, request);
-    history.push('/app/custody/requests');
+    switch (accountType) {
+      case AccountType.REGULAR:
+        const accountRequest: RequestOpenAccount = {
+          accountId: {
+            signatories: {
+              textMap: { [service.payload.provider]: {}, [service.payload.customer]: {} },
+            },
+            label: accountName,
+            version: '0',
+          },
+          observers: [publicParty],
+          ctrls: [service.payload.provider, service.payload.customer],
+        };
+        await ledger.exercise(Service.RequestOpenAccount, service.contractId, accountRequest);
+        history.push('/app/custody/requests');
+      case AccountType.ALLOCATION:
+        const nomineeIdentity = identities.find(i => i.payload.customer === accountNominee);
+        console.log('going...');
+        if (!nomineeIdentity) return;
+        const request: RequestOpenAllocationAccount = {
+          accountId: {
+            signatories: {
+              textMap: { [service.payload.provider]: {}, [service.payload.customer]: {} },
+            },
+            label: accountName,
+            version: '0',
+          },
+          // observers: { textMap: { publicParty } },
```

Comment:
`[publicParty]` ?

Reply:
Ah...was a weird type issue with `Set`. I decided to not include the public party as an observer anyway here.